src/index.html

```diff
@@ -1,13 +1,440 @@
+                  <li class="card__item">
+                    Lossless Digital Audio Transmission
+                  </li>
+                  <li class="card__item">Easy & Stable Stereo Pairing</li>
+                  <li class="card__item">
+                    Crisp and Clear High Frequency Sound
+                  </li>
+                  <li class="card__item">
+                    Streams from Cloud Music and Local Library
+                  </li>
+                  <li class="card__item">
+                    Auto Music Playback from Last Song Stopped
+                  </li>
+                </ul>
+              </div>
+              <div class="features__card features__card2">
+                <div class="features__card_title">Connectivity</div>
+                <ul class="features__card_list">
+                  <li class="card__item">Hands Free Wireless Audio</li>
+                  <li class="card__item">Bluetooth 4.0 LE</li>
+                  <li class="card__item">Wi-Fi 2.4 GHz (802.11 b/g/n)</li>
+                  <li class="card__item">Smart Multiroom System Set Up</li>
+                  <li class="card__item">
+                    Party Mode with 6.0 Units and above
+                  </li>
+                  <li class="card__item">
+                    Powerful MESHNET Multi Speaker Network
+                  </li>
+                </ul>
+              </div>
+              <div class="features__card features__card3">
+                <div class="features__card_title">App Features</div>
+                <ul class="features__card_list">
+                  <li class="card__item">Customize Music Schedule</li>
+                  <li class="card__item">Wake Up with Favorite Songs</li>
+                  <li class="card__item">Home Detection Auto Wake Up</li>
+                  <li class="card__item">Color Wheel & Saturation Change</li>
+                </ul>
+              </div>
+            </div>
+            <img
+              src="/images/features/rectangle.svg"
+              alt=""
+              class="features__rectangle_image"
+            >
+            <img
+              src="/images/features/features__image.svg"
+              alt=""
+              class="features__image_image"
+            >
+            <div class="features__arrows">
+              <img
+                src="/images/features/left.svg"
+                alt=""
+                class="features__arrow1"
+              >
+              <img
+                src="/images/features/right.svg"
+                alt=""
+                class="features__arrow1"
+              >
+              <br>
+              01 / 03
+            </div>
+          </div>
+        </section>
+        <section class="questions">
+          <div class="questions__container">
+            <div class="questions__title">DO YOU HAVE ANY QUESTIONS?</div>
+            <div class="questions__form">
+              <form action="get"
```

Comment:
```suggestion
              <form action="get"
```
атрибу action повинен вказувати адресу, куди ми відправляємо дані з форми. Коли в тебе прописано get, то ми отримуємо 404 помилку при сабміті. Виправ це
get - це метод